client/src/main/com/sinch/sdk/auth/adapters/VerificationApplicationAuthManager.java

```diff
@@ -42,7 +41,7 @@ public void resetToken() {
 
   @Override
   public Collection<Pair<String, String>> getAuthorizationHeaders(
-      String method, String httpContentType, String path, String body) {
+      String timestamp, String method, String httpContentType, String path, String body) {
```

Comment:
You may remove the `toString()` when setting the x-timestamp header at line https://github.com/sinch/sinch-sdk-java/pull/26/files#diff-feb9f88279fca9ebde2e5c47af9da62694ad1d41b00f3e6f5f3672d6b51e3fa2L63